### PR TITLE
share my ortho_4x12 layout symbolic

### PR DESCRIPTION
**Pull request by leico**
Sharing my key layout.

### CODE REVIEW

File: layouts/community/ortho_4x12/symbolic/config.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_SYMBOLIC_LAYOUT_H
+#define CONFIG_SYMBOLIC_LAYOUT_H
+
+#include QMK_KEYBOARD_CONFIG_H
```
Comment (drashna) [inline layouts/community/ortho_4x12/symbolic/config.h]: This line isn't actually needed, it doesn't do anything.

File: layouts/community/ortho_4x12/symbolic/config.h [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_SYMBOLIC_LAYOUT_H
```
Comment (drashna) [inline layouts/community/ortho_4x12/symbolic/config.h]: If you could replace these two lines with `#pragma once`, and remove the endif, at the end?

File: layouts/community/ortho_4x12/symbolic/keymap.c [hunk 3]
```diff
@@ -0,0 +1,173 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```
Comment (drashna) [inline layouts/community/ortho_4x12/symbolic/keymap.c]: These two includes aren't actually required.  You can safely remove them, ans the `QMK_KEYBOARD_H` include should actually cover them.

**Comment by drashna** [comment]
Thanks!

**Comment by leico** [comment]
Thank you for review. Now typing with this layout / vitamins_included. Perhaps no problem it works.

**Comment by drashna** [comment]
Glad to hear it.

And there was no issues to prevent it from working, We just want to keep the code consistent and clean, for the most part.

**Pull request merged**